website/docs/components/table/partials/specifications/anatomy.md

```diff
@@ -0,0 +1,11 @@
+## Anatomy
```

Comment:
This probably also needs the cell anatomy and to be organized into a table, but can be added in a follow-up PR

Reply:
If you want to export/upload the Figma images, that would be awesome